ruby/lib/minitest/queue/statsd.rb

```diff
@@ -0,0 +1,65 @@
+# Small, limited StatsD implementation.
+# We can't use the standard statsd-instrument for instrumenting our buildkite setup, because:
```

Comment:
These comments are very weird considering we're in an open source gem.

We should rewrite them in a more general way.